components/cache/adapters/array_cache_adapter.rst

```diff
@@ -0,0 +1,20 @@
+.. index::
+    single: Cache Pool
+    single: Array Cache
+
+Array Cache Adapter
+===================
+
+This adapter is only useful for testing purposes because contents are stored in
+memory and not persisted in any way. Besides, some features explained later are
+not available, such as the deferred saves::
```

Comment:
Used your re-write to expand a bit more, to:
```
Generally, this adapter is useful for testing purposes, as its contents are stored in memory
and not persisted outside the running PHP process in any way. It can also be useful while
warming up caches, due to the :method:`Symfony\\Component\\Cache\\Adapter\\ArrayAdapter::getValues()`
method.

This adapter can be passed a default cache lifetime as its first parameter, and a serialization
toggle as its second parameter::
```